src/Controller.js

```diff
@@ -38,38 +46,45 @@
     },
 
     renderShip(ship) {
-      // const ship = this.ship;
+      //const ship = this.ship;
 
       const shipPortIndex = ship.itinerary.ports.indexOf(ship.currentPort);
-      const portElement = document.querySelector(`[data-port-index= '${shipPortIndex}']`);
+      const portElement = document.querySelector(
+        `[data-port-index= '${shipPortIndex}']`
+      );
 
-      const shipElement = document.querySelector('#ship');
-      shipElement.style.top = `${portElement.offsetTop}px`;
-      shipElement.style.left = `${portElement.offsetLeft}px`;
+      const shipElement = document.querySelector("#ship");
+      //shipElement.style.top = `${portElement.offsetTop}px`;
+      //shipElement.style.left = `${portElement.offsetLeft}px`;
```

Comment:
For all cases where you have commented out code -  try not to commit with commented out code. If it doesn't do anything it probably shouldn't be here.Also don't be afraid of deleting your code if you don't need it anymore, you wrote it once, you are. more than capable of writing it again :) (also you can always go back in the commit history to check/retrieve what you had in the past).
If you keep commented out code for reference, it's better to leave a comment like:
`// Set ship in the right position with offset values`

General note on comments in the code: comment why you're doing something, not what are you doing (what should be clear, and in this case is, from the code). Comments should explain your intentions. Writing good comments is difficult though :) I'm still having a problem sometimes to write a meaningful message, despite of hundreds/thousands of them I've already written in the past :P